taichi/transforms/lower_access.cpp

```diff
@@ -142,7 +149,12 @@ class LowerAccess : public IRVisitor {
         auto lookup = lowered.push_back<SNodeLookupStmt>(
             snode, last, linearized, needs_activation);
         int chid = snode->child_id(snodes[i + 1]);
-        last = lowered.push_back<GetChStmt>(lookup, chid);
+        if (is_bit_vectorized && snode->type == SNodeType::dense &&
+            i == length - 2) {
+          last = lowered.push_back<GetChStmt>(lookup, chid, true);
```

Comment:
core change: to fix `GetChStmt`'s type, tag the last generated `GetChStmt` with `is_bit_vectorized=true`